packages/manager/apps/dedicated/client/app/components/ip/agoraOrder/ipv4/ipv4.html

```diff
@@ -42,6 +43,14 @@
                             aria-hidden="true"
                         ></span>
                     </label>
+                    <div style="height:24px;">
```

Comment:
to define in css file

Reply:
To me it's not necessary to create a new file only for this property which won't be reused on an other component